data/_ui/messages.txt

```diff
@@ -0,0 +1,161 @@
+# Copyright (c) 2025 by TomGoodIdea
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+
+# Categories:
+
+# The category used for most important system messages.
+"message category" "high"
+	"main color" "message importance highest"
+	"log color" "message log importance highest"
+	important
+
+# The default category used for most engine messages, including hails.
+"message category" "normal"
```

Comment:
Changing `high` to `normal` makes sense, but now we have both `normal` as category and `high` as color name.
Sometimes it is better to stick with the terminology that was already used, because half a change is likely more confusing than to just keep calling this category `high`.

Reply:
I'd just rename the colors. It could break UI plugins, but they should be kept up-to-date anyway, so I guess backward compatibility isn't that important in this case.